lte/gateway/c/session_manager/SessionState.cpp

```diff
@@ -2345,6 +2430,14 @@ void SessionState::clear_create_session_response() {
   create_session_response_ = CreateSessionResponse();
 }
 
+StatsPerPolicy SessionState::get_policy_stats(std::string rule_id) {
```

Comment:
Maybe return `optional<StatsPerPolicy>` to better differentiate not finding a record vs 0 record

Reply:
(unless you want to create an empty record when it's not found)

Reply:
Its just for testing so I think empty on not found is fine